packages/core/.storybook/main.js

```diff
@@ -14,13 +12,13 @@ module.exports = {
 					loader: "sass-loader",
 					options: {
 						sassOptions: {
-							includePaths: [path.resolve(__dirname + "../src")]
-						}
-					}
-				}
-			]
+							includePaths: [path.resolve(__dirname + "../src")],
+						},
+					},
+				},
+			],
```

Comment:
Ew, trailing commas

Reply:
I think those can disabled through prettier configs.

IMO we shouldn't care about any of this stuff, they're all arbitrary preferences that are another reason why Prettier exists